collection-scripts/gather_kube_apiserver_termination_logs

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+TERMINATION_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/kube_apiserver_termination_logs"}
+
+mkdir -p "${TERMINATION_LOG_PATH}"/
+
+function get_dump_off_node {
+    local debugPod=""
+    
+    #Get debug pod's name
+    debugPod=$(oc debug --to-namespace="default" node/"$1" -o jsonpath='{.metadata.name}')
+
+    #Start Debug pod force it to stay up until removed in "default" namespace
+    oc debug --to-namespace="default" node/"$1" -- /bin/bash -c 'sleep 300' > /dev/null 2>&1 &
+
+    #Mimic a normal oc call, i.e pause between two successive calls to allow pod to register
+    sleep 2
+    oc wait -n "default" --for=condition=Ready pod/"$debugPod" --timeout=30s
+
+    if [ -z "$debugPod" ]
+    then
+      echo "Debug pod for node ""$1"" never activated"
+    else
+      #Copy API watch log
+      echo "Copying termination log on node ""$1"""
+      oc cp  --loglevel 1 -n "default" "$debugPod":/host/var/log/kube-apiserver/termination.log "${TERMINATION_LOG_PATH}"/"$1"_termination.log > /dev/null 2>&1 && PIDS+=($!)
```

Comment:
Is this attempting to do https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_audit_logs#L16 ?

Reply:
This PR isn't gathering the audit logs. It is gathering the watch termination logs from the API server.

Reply:
Why wouldn't `oc adm node-logs --role=master --path="$path"` do the job?